src/components/htmlvideoplayer/plugin.js

```diff
@@ -1378,6 +1378,10 @@ define(['browser', 'require', 'events', 'apphost', 'loading', 'dom', 'playbackMa
             }
         }
 
+        function enableCustomVideoControls() {
+            return true;
+        }
```

Comment:
heh, I want some explanation here... you're basically always default to `else` clause, which (as I judge from comments) could break on WebOS which have old-ish Chrome bundled.

Reply:
Seems like this might be resolved by simply removing the CSS rule.

Reply:
Those things there were for a reason, so I'm sensing that we'd need them for some edge cases...

Reply:
I specifically mean the four lines of CSS he removed, which look like they would globally hide all custom video controls on firefox.

Reply:
> 
> 
> heh, I want some explanation here... you're basically always default to `else` clause, which (as I judge from comments) could break on WebOS which have old-ish Chrome bundled.

i am limited here